.github/workflows/release-process.yml

```diff
@@ -37,17 +37,22 @@ jobs:
           ./hack/build-release.sh
           cd cli
           ./hack/build-binaries.sh
-          shasum -a 256 kctrl-* | tee -a ../tmp/checksums.txt
+          shasum -a 256 ./kctrl-* | tee -a ../tmp/checksums.txt
+          echo "# :open_file_folder: Files Checksum" | tee ../tmp/checksums-formatted.txt
+          echo \`\`\` | tee -a ../tmp/checksums-formatted.txt
+          cat ../tmp/checksums.txt | tee -a ../tmp/checksums-formatted.txt
+          echo \`\`\` | tee -a ../tmp/checksums-formatted.txt
```

Comment:
I tried something like `{ echo "# :open_file_folder: Files Checksum"; cat ../tmp/checksums.txt } tee ../tmp/checksums-formatted.txt` to make it inline, but it didn't work expectedly.

Reply:
nit: You could replace the escape characters of your echo with

echo '```'

It's a little more readable to me. But this ain't a blocker :)

Reply:
Done